src/docs/Contributing/Designers/AddNewIcons/index.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Add new icons'
+order: 4
+---
+
+> First of all, each icon must be reviewed by the designer(s) in charge of the quality and the consistency of the icons at LEROY MERLIN. Second of all, each new icon must be declined in the following sizes: `24px`, `32px`, `48px` & `64px`. Here are the steps that you must follow to add new icons in Mozaic.
+
+## 1. Get in touch with Mozaic's team
+
+It's important to discuss your needs and what can be produced together before starting to create the said icon. Our icon designer is [Simon Roger](mailto:simon.roger@ext.leroymerlin.fr) for now. Get in touch with him to speak about format and sizes of your icon(s).
+
+## 2. Icon design
+
+This is the point where the icon is designed either by you or our design team.
+
+## 3. New branch on GitHub
+
+You must create a branch to be able to push your work in Mozaic's design system repository. We advise you to name it like this `YYWW-icons` (1949-icons) to be meaningful and consistent.
```

Comment:
Concerning branch naming, there is this page on Git conventions :
=> http://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#branch-names 

Maybe we should try to propose a convention that is close to it?